DirectProgramming/DPC++FPGA/Tutorials/Features/device_global/src/device_global.cpp

```diff
@@ -0,0 +1,112 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <math.h>
+
+#include <CL/sycl.hpp>
```

Comment:
```suggestion
#include <sycl/sycl.hpp>
```
This is a recent compiler change, so we are making this change to all samples, here is the associated pending PR for the existing samples: https://github.com/oneapi-src/oneAPI-samples/pull/1072